lib/serializers/repp/domain.rb

```diff
@@ -15,7 +15,7 @@ def to_json(obj = domain)
           updated_at: obj.updated_at, expire_time: obj.expire_time, outzone_at: obj.outzone_at,
           delete_date: obj.delete_date, force_delete_date: obj.force_delete_date,
           contacts: contacts, nameservers: nameservers, dnssec_keys: dnssec_keys,
-          statuses: obj.statuses, registrar: registrar
```

Comment:
I removed because `status_notes` has the statuses as hash keys and notes as values. So we have both data
@vohmar liked showing this way. But I can change. @OlegPhenomenon @vohmar which way do you suggest and prefer to use in this case?